### ISSUE TITLE
Fucky interactions with non-humanoids and the apocalypse rune hallucination thing

### ISSUE THREAD

**Issue by DaydreamIQ** [tgstation/tgstation#77714]
Reporting client version: 514.1589

## Round ID:
[212771](https://scrubby.melonmesa.com/round/212771)

## Testmerges:
- [Trying to qdelete a callback now lags the server instead.](https://www.github.com/tgstation/tgstation/pull/77447)
- [Revert "Bumps rust-g to `3.0.0` "](https://www.github.com/tgstation/tgstation/pull/77307)
- [MODLink System (+ NWTLMM)](https://www.github.com/tgstation/tgstation/pull/77639)

## Reproduction:
![image](https://github.com/tgstation/tgstation/assets/62606051/a9863e90-3505-4e13-95f6-2bf7bde7473a)
So basically the weird overlay the apocalypse rune grants normally makes people look like cultists, but if they're a mob like say a construct or a simplemob. They just turn completely invisible. Unsure if this is intended because apocalypse runes are never used

**Comment by Sealed101** [comment]
it's probably the same situation as with the bubblegum's mayhem in a bottle - the slaughter demon overlays there aren't placed to the laying position when the mob dies. not sure if it's intended

**Comment by Sealed101** [comment]
however the mobs becoming invisible is a bug yeah
